src/nitserial.nit

```diff
@@ -155,13 +157,47 @@ for mmodule in mmodules do
 		module_path += ".nit"
 	end
 
+	var target_modules = null
+	var importations = null
+	var mgroup = mmodule.mgroup
+	if toolcontext.opt_depth.value == 1 then
+		# group
+		if mgroup != null then target_modules = mgroup.mmodules
+	else if toolcontext.opt_depth.value == 2 then
+		# project
+		target_modules = new Array[MModule]
+		importations = new Array[MModule]
+		if mgroup != null then
+			for g in mgroup.mproject.mgroups do
+				target_modules.add_all g.mmodules
+			end
+
+			for g in mgroup.in_nesting.direct_smallers do
+				var dm = g.default_mmodule
+				if dm != null then
+					importations.add dm
```

Comment:
It tells the generated module to import the default entrypoint of all groups of the project.